src/pages/publication/description_tab/projects_field/AutocompleteTextField.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC, KeyboardEvent } from 'react';
+import { TextField, CircularProgress, TextFieldProps } from '@material-ui/core';
+import { AutocompleteRenderInputParams } from '@material-ui/lab';
+import styled from 'styled-components';
+import SearchIcon from '@material-ui/icons/Search';
+
+const StyledSearchIcon = styled(SearchIcon)`
+  margin-left: 0.5rem;
+  color: ${({ theme }) => theme.palette.text.disabled};
+`;
+
+interface AutocompleteTextField extends AutocompleteRenderInputParams, Pick<TextFieldProps, 'placeholder'> {
+  isLoading: boolean;
+  dataTestId?: string;
+}
+
+export const AutocompleteTextField: FC<AutocompleteTextField> = ({ isLoading, dataTestId, ...params }) => (
+  <TextField
+    {...params}
+    variant="outlined"
+    fullWidth
+    onKeyDown={(event: KeyboardEvent) => {
+      if (event.key === 'Backspace') {
+        // Disable removing chips with backspace
```

Comment:
why though?

Reply:
![bilde](https://user-images.githubusercontent.com/6313468/96596496-12639f80-12ed-11eb-9cca-f9eef12950b3.png)
hvis man holder inn backspace for å fjerne søkestrengen en tidel for lenge så har du plutselig fjernet alle prosjektene man har lagt inn også

Reply:
men standard er vel at man kan backspace'e bort tags? jeg synes ikke vi bør bryte med default behavior.

Reply:
bare se hvordan det fungerer på Jira f.eks.

Reply:
hm ja, de har faktisk både fjern-knapp og backspace ja...
jeg ble i hvert fall irritert noen ganger fordi alt plutselig ble visket ut når jeg bare ville fjerne søkestreng, men det er kanskje bare meg det da :p 

![bilde](https://user-images.githubusercontent.com/6313468/96598569-493ab500-12ef-11eb-802c-96cdcda63a0b.png)
da bør vi kanskje flytte søkeikon litt slik at cursor faktisk er rett bak tags da, og ikke fixed til etter søkeikon? nå ser det ikke logisk ut at prosjekt fjernes med backspace imo

Reply:
ser ikke ut som det er bare-bare å ha ikon i placeholder 🤔

Reply:
Trenger vi å ha søkeikonet der i det hele tatt etter man har lagt til et prosjekt da? Kan vi ikke ha det kun i starten? også er det kun cursor ellers? slik som i Jira?

Reply:
kan prøve det 👍